projects/i18n/languages/ukrainian/kit.ts

```diff
@@ -0,0 +1,49 @@
+import {LanguageKit} from '@taiga-ui/i18n/interfaces';
+
+export const TUI_UKRAINIAN_LANGUAGE_KIT: LanguageKit = {
+    cancel: 'Скасувати',
+    done: 'Готово',
+    more: 'Ще',
+    otherDate: 'Інша дата ...',
+    mobileCalendarTexts: ['Виберіть день', 'Виберіть період'],
+    range: ['від', 'до'],
+    countTexts: ['Плюс', 'Мінус'],
+    time: {
+        'HH:MM': 'ГГ:ХХ',
+        'HH:MM:SS': 'ГГ: ХХ: СС',
```

Comment:
let's remove whitespaces here and in the next line. These whitespaces could break input-time masks